migrations/Version20231107204142.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+class Version20231107204142 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'Add column remote_followers_count';
+    }
+
+    public function up(Schema $schema): void
+    {
+        $this->addSql('ALTER TABLE "user" ADD ap_followers_count INT DEFAULT NULL');
```

Comment:
If you add new columns, try to add them to the entity in `src/Entity/<class>.php` instead, to avoid DB migration diff issues between Doctrine and the DB.

Reply:
You added it to ActivityPubActorTrait instead I see? hm. OK. not sure if that is good or bad 😖

Reply:
It is added there through the `ActivityPubActorTrait`. Didn't know that that would lead to an update (does it?)

Reply:
Yes it's used here: https://github.com/MbinOrg/mbin/blob/239071ad2822acc3f05273da5df8a238343af9ac/src/Entity/User.php#L44 I think this is expected behavior. Assuming this is what we want.

After the main branch changes merged into this branch (which I just did recently). You can verify yourself by executing: `php bin/console doctrine:migrations:diff`

If this result into a new file with additional up and down queries.. well it shouldn't, since then you forgot still something.

Reply:
```
docker compose exec php php bin/console doctrine:migrations:diff

In NoChangesDetected.php line 13:
                                                    
  No changes detected in your mapping information.  
                                                    

doctrine:migrations:diff [--configuration CONFIGURATION] [--em EM] [--conn CONN] [--namespace NAMESPACE] [--filter-expression FILTER-EXPRESSION] [--formatted] [--line-length LINE-LENGTH] [--check-database-platform [CHECK-DATABASE-PLATFORM]] [--allow-empty-diff] [--from-empty-schema]
```